source/includes/api/_areas_v2.md

```diff
@@ -221,53 +293,70 @@ curl -X POST https://api.resourcewatch.org/v2/area
-webhookUrl           | Webhook URL to be provided to the subscription (only used in case the email is not set).           | String  | https://www.google.com/ |
-status               | The status of the area - either 'saved' or 'pending'. Read-only attribute.                         | String  | 'saved'    |
-subscriptionId       | The ID of the subscription associated with this area. Read-only attribute.                         | String  | 5e4d7c47dd8fa31290d548ae |
+* `geostore` with the ID of a geostore object if you are creating an area that references a geostore;
+* `wdpaid` with the ID of a protected area if you are creating an area that references a protected area;
+* `iso` object with a valid country/region/subregion if you are creating an area that references an admin country/region/subregion;
+* `use` object with valid id and name of a land use concessioned area, if you are creating an area that references a land use area.
 
-### Email notification
+Please check the [Area model reference](/index-rw.html#area-reference) for details on what values each field is expected to provide.
 
 According to multiple factors (including the `geostore` that is associated with the area, if the area subscribes to `fireAlerts`, `deforestationAlerts`, etc.), there might be a period of time in which the data for the area is being generated. While that is the case, the area will have `status` set to `'pending'`. Once the area data is ready, the `status` of the area will be updated to `'saved'`.
 
-After creating an area, if the `email` field of the area has a valid email, an email is sent to the user. The email content varies according to the status of the area:
+### Errors for creating an area
+
+Error code     | Error message (example)     | Description
+-------------- | --------------------------- | ------------------------------------------------------------
+400            | `<field> can not be empty.` | You are missing a required field while creating the area.
+400            | `<field> is not valid.`     | You provided an invalid field while creating the area.
+401            | `Unauthorized`              | No token was provided.
+
+### Email notifications
+
+After creating an area, if the `email` field of the area contains a valid email, an email is sent to the user. The email content varies according to the status of the area:
 
 * If the area has status `saved`, an email is sent to let the user know the area of interest is ready to be viewed.
 * If the area has status `pending`, an email is sent to let the user know the area of interest is being generated and will be available later.
 
-### Email substitution parameters
+Emails are sent using [Sparkpost](https://www.sparkpost.com/) - so, for the emails to be sent, there should exist templates in Sparkpost ready to be sent, taking into account the different languages supported by the Areas service:
```

Comment:
Is this relevant to an API user? do they have access to sparkpost, or need to worry about it in any way?

Reply:
This is relevant to API developers, so moved to the developer docs. Also added some information on getting access to the Sparkpost account.